src-v2/Judge/NTS.Judge.MAUI/wwwroot/css/app.css

```diff
@@ -3,6 +3,19 @@
     margin-bottom: 5px;
 }
 
+.action-column{
+    display:inline;
+    margin-left:20px;
+}
+
+.half-width{
+    width:45%;
+}
+
+.children-component{
+    margin-top:10px;
+}
+
```

Comment:
Can't we use Mud's styling framework instead of custom styles?

Reply:
Some styles can be set with Mud styling.
Like `margin-top:10px;` but the general idea here is to set a class which can target an entire component or apply styles that are missing from Mud as a semantic property, e.g. `half-width` should ensure that two components with the class will fit on the same line.
I suggest we keep these class configurations until the focus comes to the UI fine tuning.